packages/flutter/lib/src/foundation/core_io.dart

```diff
@@ -1,49 +1,101 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
+// Copyright 2019 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
 import 'dart:async';
 import 'dart:developer' show Timeline, Flow;
+import 'dart:io';
 import 'dart:isolate';
 
 import 'package:meta/meta.dart';
 
+import 'assertions.dart';
 import 'constants.dart';
+import 'core_stub.dart' as core;
+import 'platform.dart';
 
-/// Signature for the callback passed to [compute].
-///
-/// {@macro flutter.foundation.compute.types}
-///
-/// Instances of [ComputeCallback] must be top-level functions or static methods
-/// of classes, not closures or instance methods of objects.
-///
-/// {@macro flutter.foundation.compute.limitations}
-typedef ComputeCallback<Q, R> = FutureOr<R> Function(Q message);
+/// The io implementation of defaultTargetPlatform.
```

Comment:
Should this use `//` instead of `///`? It's not high-quality dartdocs.
Or maybe we should use templates/macros?